scout/adapter/mongo/case.py

```diff
@@ -574,12 +574,16 @@ def nr_cases(self, institute_id=None):
         Returns:
```

Comment:
I wonder if this query can be entirely skipped. One could sum up the numbers returned by [this other function](https://github.com/Clinical-Genomics/scout/blob/aecafa3219e3be066dbc137896d3d1b603dcd144/scout/server/blueprints/institutes/controllers.py#L420) (that is necessary) instead?

Reply:
Yes, I thought the same. But it should be really quick, and then we have to ensure the others are not overlapping, and the function is not used elsewhere.

Reply:
Ok, does not seem to be used elsewhere, and the `$group` should make it as non-overlapping as it gets.

Reply:
Yes, I checked, it's used only in there

Reply:
Heh, it was surprisingly slightly slower when removing it; possibly my slower python juggling of dict-array conversions than the rather to the point db summing, or just a different day. But both still clocked faster than the original.

![Screenshot 2023-04-18 at 10 44 10](https://user-images.githubusercontent.com/758570/232723946-e785a45b-fd4d-437a-81b8-62c29e2fd00a.png)

I would still keep the latest without the double db request, just to keep things simple, and we see how it performs on a large db?

Reply:
Ok!